src/components/v5/common/Filter/types.ts

```diff
@@ -36,6 +36,7 @@ export type FilterOptionProps = {
   id: NestedFilterOption;
   title: Message;
   icon?: JSX.Element;
+  children?: FilterOptionProps[];
```

Comment:
I don't know if children is a good name, maybe something like `nestedOptions` or `subOptions`.

Reply:
changed